### ISSUE TITLE
Vite plugins (e.g. Babel) not executed for entrypoints

### ISSUE THREAD

**Issue by dword-design** [wxt-dev/wxt#504]
### Describe the bug

Vite plugins aren't executed for entrypoints. Reason is probably because they are loaded via jiti. I like to use some babel plugins for my projects, so it would be great if entrypoints are processed regularly via Vite.

### To Reproduce

```js
// wxt.config.js

import { defineConfig } from 'wxt';
import babel from 'vite-plugin-babel'

export default defineConfig({
  vite: () => ({
    plugins: [babel()],
  }),
});
```

```js
// entrypoints/background.js

import map from 'lodash/fp/map'

// use some babel plugin here
export default defineBackground(() => console.log([1, 2] |> map(x => x * 2))
```

### Expected behavior

The code runs through.

### Actual behavior

Builds exits with a parsing error.

### Additional context

It works with `vite-plugin-web-extension` and `Nuxt`.

**Comment by aklinker1** [comment]
Yeah, for this to work we'd have to use vite to transpile the entrypoints. I will look into this after some of the other large issues are handled.

For now, you can use custom plugins in other files, just not the entrypoints.

**Comment by aklinker1** [comment]
I'm gonna close this issue, it will be fixed in #506, follow that one for updates.
